ifrs17/Import/4ImportScope-TechnicalMargin.ipynb

```diff
@@ -236,7 +236,7 @@
                 "\n    double Value => GetScope<ValidAmountType>(Identity.DataNode).BeAmountTypes",
                 "\n                       .Except(GetStorage().GetNonAttributableAmountType())",
                 "\n                       .Sum(at => GetScope<PresentValue>((Identity, at, EstimateTypes.BE, (int?)null), o => o.WithContext(EconomicBasis)).Value) +",
-                "\n                    GetScope<RaLocked>(Identity).Value;",
+                "\n                       GetScope<PresentValue>((Identity, (string)null, EstimateTypes.RA, (int?)null), o => o.WithContext(EconomicBasis)).Value;",
```

Comment:
I did not kill RaLocked. But made explicit here the choice of EconomicBasis so that it responds to VFA.

Lic does not compute any csm and loss from NB 5ImportScope-ToIfrsVar where we call the scopes to be computed:
![image](https://user-images.githubusercontent.com/103409906/236409768-9eb52f28-75f8-4523-949f-c969d2100004.png)